### DISCUSSION TITLE
What is Parser-16?

### DISCUSSION THREAD

**Discussion by AjayChambers** [microsoft/pylance-release#1639] [Q&A]
Dealing with a `parser-16` error, and was wondering if anyone actually knows what Parser-16 actually is (obviously its a parser, but what does the 16 mean)? And what does it mean when I receive a parser-16 error?

**Comment by jakebailey** [comment]
"parser-16" errors are emitted by the old unmaintained "Microsoft" language server, not Pylance, so you may want to check your language server configuration to switch to Pylance.
